specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+  ],
+  "produces": [
+    "application/json;odata=nometadata"
+  ],
+  "paths": {
+    "/Tables": {
+      "get": {
+        "tags": [
+          "table"
+        ],
+        "operationId": "Table_Query",
+        "x-ms-examples": {
+          "TableGet": {
+            "$ref": "./examples/TableGet.json"
+          }
+        },
+        "description": "Queries tables under the given account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/DataServiceVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/formatParameter"
+          },
+          {
+            "$ref": "#/parameters/topParameter"
+          },
+          {
+            "$ref": "#/parameters/selectParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, table query completed.",
+            "headers": {
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the Queue service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+              },
+              "x-ms-continuation-NextTableName": {
+                "type": "string",
+                "description": "This header contains the continuation token value."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/TableQueryResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "table"
+        ],
+        "operationId": "Table_Create",
+        "x-ms-examples": {
+          "TableCreate": {
+            "$ref": "./examples/TableCreate.json"
+          }
+        },
+        "description": "Creates a new table under the given account.",
+        "parameters": [
```

Comment:
Can we add support for the `Prefer` header?
https://docs.microsoft.com/en-us/rest/api/storageservices/setting-the-prefer-header-to-manage-response-echo-on-insert-operations